### DISCUSSION TITLE
Settings examples

### DISCUSSION THREAD

**Discussion by Ev-Bro** [Extraltodeus/multi-subject-render#6] [Ideas]
This is very cool. Some more examples would be very helpful!

**Comment by Extraltodeus** [comment]
The settings shown in the readme of that repository are actually quite not bad! The hardest setting is the "Foreground depth cut treshold". Which I would put around ~50 for 2D/painting stuff and 90-98 for photorealistic things.

**Comment by Arumizu** [comment]
I've been messing around with the recommended settings, and here are some of the images I've managed to create (keeping in mind my SD is tuned towards anime):

![00032](https://user-images.githubusercontent.com/37074124/204118918-8ade4b84-3962-4b79-85ea-e10aaedce792.png)
![00033](https://user-images.githubusercontent.com/37074124/204118924-9fd68b3b-0f7f-4a50-8b9c-379f51f071e9.png)
![00034](https://user-images.githubusercontent.com/37074124/204118928-eec02926-6a6e-4edd-b86d-b5728f5f73d1.png)

I noticed that the foreground prompts started to deteriorate and discard some of the first two characters' features that were prompted (the eyes being one of the main details). 

Here's my settings for reference:
`cyberpunk, neon lights, wide shot
Negative prompt: (bad_prompt_version2:0.8), lowres, bad anatomy, bad hands, text, error, cropped, worst quality, low quality, normal quality, jpeg artifacts, signature, watermark, username, blurry, artist name, (3d:1.3), (simple background:1.5)
Steps: 12, Sampler: DPM++ 2S a Karras, CFG scale: 7, Seed: 2427706542, Size: 1152x768, Model hash: 714316a8, Hypernet: liya, Hypernet strength: 0.851, Denoising strength: 0.6, Clip skip: 2`

One thing to note is that my foreground image generation is using 448x640, then upscaled in the final blend.

MSR Settings: 
(Foreground):
`1girl, highres, (HDR:1.3), cowboy shot, (pink hair with long ponytail:1.2), (light green eyes:1.3), (small female breasts:1.2), (white parka:1.1), (black bodysuit under clothes:1.3), skin tight, covered navel, shy, (detailed eyes:1.1), (blush:1.1), (clean_hands:0.6)`
`1girl, highres, (HDR:1.3), cowboy shot, (blue hair:1.2), (blue eyes:1.3), casual outfit, light smile, (detailed eyes:1.1), (blush:1.1), (clean_hands:0.6)`
`1girl, highres, (HDR:1.3), cowboy shot, (blonde hair:1.2), (brown eyes:1.3), cyberpunk outfit, surprised, (detailed eyes:1.1), (blush:1.1), (clean_hands:0.6)`

`Number of foreground images: 3, 
foreground steps: 12, 
foreground cfg scale: 7, 
foreground new seed+: 1000, 
foreground sampler: DPM++ 2S a Karras, 
Clip skip: 2, 
Foreground width/height: 64`

(Final Blend)
`cyberpunk, neon lights, wide shot blend steps: 12, cfg scale: 7, denoising strength: 0.6, sampler: DPM++ 2S a Karras, width: 1152, height: 768, depth cut threshold: 50, face correction disabled`

Hopefully this helps a little! (edit: cleaner foreground prompt spacing + i realized i had the wrong prompt)
